src/components/MobileControls/MobileControls.tsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import React, {memo, useMemo} from 'react';
+import {Globe, Sun} from '@gravity-ui/icons';
+import block from 'bem-cn-lite';
+import allLangs from 'langs';
+import {KeyPrefix, Namespace, TFunction} from 'react-i18next';
+
+import {ControlSizes, ControlsLayout, ControlsProps, DocSettings, Lang, Theme} from '../..';
+import {useTranslation} from '../../hooks';
+import {ListItem, OnChangeValue} from '../../models';
+
+import MobileControl from './MobileControl/MobileControl';
+import './MobileControls.scss';
+
+const b = block('dc-mobile-controls');
+
+type UseTranslationReturnType = TFunction<Namespace<string>, KeyPrefix<Namespace<string>>>;
+
+export interface MobileControlsProps {
+    controlSize: ControlSizes;
+    lang: Lang;
+    userSettings: DocSettings & ControlsProps;
+}
+
+const LEGACY_LANG_ITEMS = [
```

Comment:
we support quite a large number of languages for ui and any language for documentation, why are there these three ? have you checked that the documents in other languages ​​work as they should ?

Reply:
These are the default constants that are used in the LangControl component. I just duplicated this logic. And now I'll take it out to constants.ts

Reply:
These values are used only when the user has not added their languages to the .yfm-config